src/element.ts

```diff
@@ -491,4 +488,95 @@ export abstract class Element {
     const fontInfo = this.fontInfo;
     this.setFont(fontInfo.family, fontInfo.size, weight, fontInfo.style);
   }
+
+  /** Get element width. */
+  getWidth(): number {
+    return this.width;
+  }
+
+  /** Set element width. */
+  setWidth(width: number): this {
+    this.width = width;
+    return this;
+  }
+
+  /** Shift element down `yShift` pixels. Negative values shift up. */
+  setYShift(yShift: number): this {
+    this.yShift = yShift;
+    return this;
+  }
+
+  /** Get shift element `yShift` */
+  getYShift(): number {
+    return this.yShift;
+  }
+
+  /**
+   * Set shift element right `xShift` pixels. Negative values shift left.
+   */
+  setXShift(xShift: number): this {
+    this.xShift = xShift;
+    return this;
+  }
+
+  /** Get shift element `xShift` */
+  getXShift(): number {
+    return this.xShift;
+  }
+
+  /**
+   * Set element text
+   */
+  setText(text: string): this {
+    this.text = text;
+    return this;
+  }
+
+  /** Get element text */
+  getText(): string {
+    return this.text;
+  }
+
+  /**
```

Comment:
Inconsistent style of one-line comments. I'm OK with either as long as we are more locally consistent. :-)

In fact, I'm totally OK with the more verbose form, even for single line comments.

```js
/**
 * Here is a single-line comment.
 */
```

This allows us to easily expand the comment in the future if necessary.

Also.... this is extremely nitpicky, but I think comments that are complete sentences should end with the correct punctuation, like a period or question mark if it's a question. It just makes the code base look nicer overall.

Reply:
unyfied